README.md

```diff
@@ -1 +1,24 @@
-# couch-tracker-documentation
\ No newline at end of file
+# Couch-Tracker
+
+An Android application, to keep track of the TV shows you follow, and the movies you've watched.
+
+This repository contains all the documents and design decisions we've made.
+
+
+## Tenets
+
+- Free and open-source
+- No ads, trackers, subscriptions, etc.
+- Maintaining and developing the app can be an hobby: low (or zero) fees, low architectural complexity, supported by the community
+- User data is owned by and hosted by the respective user
+- Fast and simple
+
+
+## General overview
+
+The app stores the user data in a file, logically owned by the user. The file will be in the user partition, and could be synced with other devices (e.g. using Synthing), backed up in the cloud, or read/written by other apps that understand the same file format.
```

Comment:
> The file will be in the user partition

I agree that the app must handle this case, but we also might think of offering a more "standard" solution where the user data is in the normal app data storage and is synced by the Play Store, like most apps work. In this case there must be a button that allows you to export the file as is.

This would remove attrition for non-techincal users to use it.

Reply:
ok